root/programs/C#/Frameworks/Infrastructure/ServiceInterface/AsyncProcessingService/TMInProcessDefinition.xml

```diff
@@ -6,5 +6,5 @@
 ]>
 <!-- idの先頭には、数字を使用できない。 -->
 <TMD>
-  <Transmission id="AzureStrageCopy" assemblyName="AsyncProcess" className="AsyncProcess.ProcessCopyJob" />
+  <Transmission id="AAA" assemblyName="AsyncSvc_sample" className="AsyncSvc_sample.LayerB" />
```

Comment:
Since this is a sample, I think that you should specify the "understandable value".